integration-tests/docker/chainlink.go

```diff
@@ -0,0 +1,324 @@
+package docker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	tc "github.com/testcontainers/testcontainers-go"
+	tcwait "github.com/testcontainers/testcontainers-go/wait"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	ErrNoChainlinkImage   = errors.New("CHAINLINK_IMAGE environment variable must be set")
+	ErrNoChainlinkVersion = errors.New("CHAINLINK_VERSION environment variable must be set")
+)
+
+/* templates for CL node configuration */
+
+var (
+	NodeConfigTemplate = `RootDir = '/home/chainlink'
+
+[Database]
```

Comment:
@skudasov we also have config/config.go with similar code. Can we unify/reuse it?

Reply:
We should have it in `core` maybe, it's for the CL node.